stdlib/REPL/src/REPL.jl

```diff
@@ -451,7 +456,7 @@ mutable struct LineEditREPL <: AbstractREPL
         new(t,hascolor,prompt_color,input_color,answer_color,shell_color,help_color,history_file,in_shell,
             in_help,envcolors,false,nothing, Options(), nothing, Tuple{String,Int}[])
 end
-outstream(r::LineEditREPL) = r.t
+outstream(r::LineEditREPL) = IOContext(r.t, :LAST_SHOWN_LINE_INFOS => r.last_shown_line_infos)
```

Comment:
Don't you need to `empty!` it? Or maybe just use `Tuple{String,Int}[]` as before?

Reply:
No, `r.last_shown_line_infos` is still needed later to actually jump to the correct definition. The functions that then write to `LAST_SHOWN_LINE_INFOS` should actually `empty!` this array already.

Reply:
Apparently, this doesn't always get cleared though...

Reply:
I think you are right, this should still get emptied, because otherwise `last_shown_line_infos` is not empty when other things get printed to the REPL and the hint still gets shown.